Hydration_freenrg/bash_scripts/MinEqallsystems.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#SBATCH -n 1
+    # Request 1 gpu per job
+#SBATCH --gres=gpu:1
+    # allocate 5 processors/CPUs per GPU
+#SBATCH --cpus-per-gpu 5
+    # allocate 5 GB of memory per job
+#SBATCH --mem 5000
+    # Set job name
+#SBATCH --job-name=MinEQ
+#SBATCH --output=./scriptouts/MinEq.o
+#SBATCH --error=./scriptouts/MinEq.err
+
+source /home/matthew/mambaforge/bin/activate openbiosim
+python --version
+cd ../
+sdir="$PWD"
+reps=( rep1 rep2 rep3 )
```

Comment:
reps is not used anywhere else

Reply:
I'm not sure it matters too much at the moment, since these will initially only be run on a specific machine, but it is often good practice to run shell scripts through a tool such as [ShellCheck](https://www.shellcheck.net/) to find bugs and make sure that your script is portable.

Reply:
Also, is it possible to use a standard naming convention throughout for scripts (both shell and Python). There seems to be a mixture of `CamelCase`, `snake_case`, and other things with no spacing, e.g. `runlallsystems.sh`, or a mixture, e.g. `runonesystem_solv.sh`. (Personally, I'd prefer `snake_case` throughout.)

Reply:
The reps line has been removed and all files (both python and bash) have been changed to `snake_case`